.github/workflows/windows-ci.yml

```diff
@@ -23,6 +23,7 @@ jobs:
           php-version: '8.0'
           tools: composer:v2
           coverage: none
+          extensions: decimal
```

Comment:
It doesn't seem to be available on Windows, hence the CI failure.

Reply:
Odd that it would report:
```
==> Setup Extensions 
√ decimal Installed and enabled 
```
if it didn't actually install it. I think as long as it works on Linux though it's probably fine.